src/themes/shared/colors.ts

```diff
@@ -1,6 +1,31 @@
 export const colors = {
   white: '#FFFFFF',
   black: '#000000',
+  whiteAlpha: {
```

Comment:
Will we need these color tokens in future? I don't see that you are using them.
These two tokens `whiteAlpha` and `blackAlpha` are added it in Figma and in the figma-tokens file?
If no -> could you sync with the Designer (for adding them in Figma) and update the `figma-tokens file`?

Thank you so much 🙏🏽